src/tools/python/valexp.md

```diff
@@ -0,0 +1,44 @@
+# Values and Expressions
+
+Here is some basic Python terminology regarding data in the language.
+
+### Values
+---
+
+In Python, we represent data using *values*. These are encodings that Python recgonizes as representing some type of data. Here are some examples of different values in python:
+
+`5, 3.7, 'a', 'Hello World!, True, [1,2,3,4]`
+
+### Expressions
+---
+
+We can chain values together with *operators* (functions that can be performed on values) to create *expressions*. Here are a few examples:
+
+```py
```

Comment:
if it isn't going to return any output, use `py,norepl`  to specify python without attaching the executor